ts/Extensions/Annotations/Mixins/ControllableMixin.ts

```diff
@@ -510,8 +517,14 @@ const controllableMixin: Highcharts.AnnotationControllableMixin = {
      *
      * @param {number} dx translation for x coordinate
      * @param {number} dy translation for y coordinate
+     * @param {number|undefined} translateSecondPoint wether to translate second point
      */
-    translateShape: function (this: Highcharts.AnnotationControllable, dx: number, dy: number): void {
+    translateShape: function (
+        this: Highcharts.AnnotationControllable,
+        dx: number,
+        dy: number,
+        translateSecondPoint?: boolean
```

Comment:
The purpose of `translateSecondPoint` is not understandable unless you know the code behind it. At least the description should be improved imho.

Reply:
Thinking about it, why only a second point and not a third? Can it be changed to a number of points to translate? (with default to 1)